routes/chat_route.py

```
@@ -22,81 +22,78 @@
-        status_code=200,
     )
 
+
 @chat_router.post("/record")
-async def post(
-        user_id: str = Form(...),
-        audio_file: Optional[UploadFile] = File(None),
-        db: Session=Depends(get_db),
-        redis: Redis = Depends(get_redis_client)
+async def post_record(
+    audio_file: Optional[UploadFile] = File(None),
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)